janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -988,6 +993,60 @@ public void testRawQueries() {
         assertCount(numV / strs.length * 2, graph.indexQuery(PINDEX, "text:ducks").properties());
     }
 
+    /**
+     * Tests query parameters with raw indexQuery
+     */
+    @Test
+    public void testRawQueriesWithParameters() throws Exception {
+        if (!supportsLuceneStyleQueries()) return;
+        Parameter asc_sort_p = null;
+        Parameter desc_sort_p = null;
+
+        // ElasticSearch and Solr have different formats for sort parameters
+        String backend = config.get(ConfigElement.getPath(INDEX_BACKEND, INDEX), String.class);
```

Comment:
Try this to see if it can make it more explicit and clear. It would retrieve the default if necessary, I hope.
String backend = readConfig.get(INDEX_BACKEND, INDEX);